projects/client/RabbitMQ.Client/src/client/api/IConnectionFactory.cs

```diff
@@ -129,13 +129,6 @@ public interface IConnectionFactory
         IConnection CreateConnection(IList<string> hostnames, String clientProvidedName);
 
         /// <summary>
-        /// Advanced option.
-        ///
-        /// What task scheduler should consumer dispatcher use.
-        /// </summary>
-        TaskScheduler TaskScheduler { get; set; }
```

Comment:
If we leave this property in and mark it as `Obsolete` with an appropriate warning explaining it is no longer used we could release this as 4.1.0 and still remain SemVer-ish compatible.